src/ImmutablePropTypes.js

```diff
@@ -50,7 +50,7 @@ function getPropType(propValue) {
 function createChainableTypeChecker(validate) {
   function checkType(isRequired, props, propName, componentName, location) {
     componentName = componentName || ANONYMOUS;
-    if (props[propName] == null) {
+    if (props[propName] === null) {
```

Comment:
It's a pretty common pattern. ESLint even has two differenn options to not
complain about it in the same rule:
http://eslint.org/docs/rules/eqeqeq.html#options

>